commands/member.js

```diff
@@ -31,13 +32,14 @@ exports.role = function(message, args){
 
 exports.rmrole = function(message, args){
     let role = message.guild.roles.find(`name`, args.join(' '));
+    logger.info(`User ${message.author.username} leaving role ${role}`);
     if (role){
-    message.member.removeRole(role).catch(console.error);
-    message.channel.send(`Yor are no longer a member of: ${args.join(' ') }... \nSorry to see you go` );
-    return
+        message.author.removeRole(role).catch(logger.error);
+        message.channel.send(`Yor are no longer a member of: ${args.join(' ') }... \nSorry to see you go` );
+        return
     }
-    message.channel.send(`I can't find the role${args.join(' ')}.\nAre you sure that is the name of the role you want to remove?\nUse \`!role <role_name>\` to join a role \nUse \`!rmrole <role_name>\` to leave a role`)
-    return
+        message.channel.send(`I can't find the role${args.join(' ')}.\nAre you sure that is the name of the role you want to remove?\nUse \`!role <role_name>\` to join a role \nUse \`!rmrole <role_name>\` to leave a role`)

```

Comment:
This needs a space after role: 
message.channel.send(`I can't find the role ${args.join(' ')}.\nAre you sure that is the name of the role you want to remove?\nUse \`!role <role_name>\` to join a role \nUse \`!rmrole <role_name>\` to leave a role`)